src/util/fhir.js

```diff
@@ -80,10 +80,23 @@ function createMedicationDispenseFromMedicationRequest(medicationRequest) {
   return medicationDispense;
 }
 
+function createMedicationFromMedicationRequest(medicationRequest) {
+  let medication = {};
+  medication.resourceType = 'Medication';
+  medication.id = medicationRequest?.id + '-med';
+  if (medicationRequest.medicationCodeableConcept) {
+    medication.code = medicationRequest.medicationCodeableConcept;
+  } else if (medicationRequest.medicationReference) {
+    medication.medicationReference = medicationRequest.medicationReference;
```

Comment:
A Medication does not have a medicationReference, it needs to have it's code populated. If there is a reference in the MedicationRequest, we probably need to retrieve the Medication and get the code from that.

Reply:
Okay that was just kept in from when i copied it above, i can delete that!